draft-ietf-scitt-architecture.md

```diff
@@ -480,16 +481,16 @@ The Transparency Service provides a history of Statements, which may be made by
 
 Transparency is implemented by providing a consistent, append-only, cryptographically verifiable, publicly available record of entries.
 A SCITT instance is referred to as a Transparency Service.
-Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
-A Receipt is an offline, universally-verifiable proof that an entry is registered in the Append-only Log.
+Implementations of Transparency Services may protect their Append-only Log and Verifiable Data Structure using a combination of trusted hardware, consensus protocols, and cryptographic evidence.
+A Receipt is an offline, universally-verifiable proof that an entry is registered in the Verifiable Data Structure.
```

Comment:
Is Receipt just an alias for a VDP as defined in COMETRE? Should we update the definition accordingly and drop this line altogether?

Reply:
Is a VDP defined in CoMETER? I cant find Verifiable Data Proof here https://github.com/ietf-scitt/draft-birkholz-cose-cometre-ccf-profile/blob/main/draft-birkholz-cose-receipts-ccf-profile.md

I think Receipt is not an alias of VDP. It is a container for proof material that is verified against a VDS and is described by a profile of https://cose-wg.github.io/draft-ietf-cose-merkle-tree-proofs/draft-ietf-cose-merkle-tree-proofs.html such as COMETER. Should this be a reference to that instead ?

Reply:
agree. we already utilise the multiple proofs so its not just a theoritical thing. we found it actively useful